src/main/java/org/sagebionetworks/bridge/spring/controllers/StudyParticipantController.java

```diff
@@ -186,6 +191,24 @@ private boolean isUpToDate(DateTime modifiedSince, DateTime modifiedOn) {
         return modifiedSince != null && modifiedOn != null && modifiedSince.isAfter(modifiedOn);
     }
     
+    @PostMapping("/v5/studies/{studyid}/participants/emailRoster")
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    public StatusMessage getParticipantRoster(@PathVariable String studyId) throws JsonProcessingException {
+        UserSession session = getAuthenticatedSession(STUDY_COORDINATOR);
+
+        if (!RequestContext.get().getOrgSponsoredStudies().contains(studyId)) {
+            throw new UnauthorizedException();   
+        }
+        StudyParticipant participant = session.getParticipant();
+        if (participant.getEmail() == null || !TRUE.equals(participant.getEmailVerified())) {
+            throw new BadRequestException("A valid email address is required to send the requested participant roster.");
+        }
+        ParticipantRosterRequest request = parseJson(ParticipantRosterRequest.class);
+        participantService.getParticipantRoster(session.getAppId(), session.getId(), request);
```

Comment:
Consolidate email check? So we should pass the callerId in, or that's in the context, and do the check there.